VMware 80u2 and 80u3 updates/fixes

sureshanaparti opened the pull request:
### Description

This PR addresses power on & move datastore issues with VMware 80u2 and 80u3, and fixes tests for restore vm (disk cleanup issue), ssvm/cpvm (wait time to disconnect). Following are the details:

- **Power On issue**
lock issue during PowerOn operation (for ROOT disk, which is cloned from seeded template)

<img width="829" alt="PowerOn-LockOnDisk" src="https://github.com/user-attachments/assets/4e2cb2ae-42cf-42b3-ab9a-4d4030c8746c" />

https://knowledge.broadcom.com/external/article/316576/troubleshooting-issues-resulting-from-lo.html

Now, PowerOn operation is being tried after lock issue, and succeeds after few attempts.

- **Move datastore file issue**
access issue with move datastore file operation (when creating volume from snapshot)
<img width="927" alt="CreateVolumeFromSnapshot-MoveDiskFailed" src="https://github.com/user-attachments/assets/5aaef9eb-d531-4ad4-b3d6-6e1ea5ee18cf" />

Now, Move datastore file operation is being tried after access issue, and succeeds after few attempts.

- **Restore VM test behavior/issue**
test checks for old root disk existence and trying to cleanup.
<img width="690" alt="RestoreVM-RootDiskNotFound" src="https://github.com/user-attachments/assets/fb845bc1-1fc0-495f-b435-2d3131bb51d6" />

 But for VMware, old root disk is force expunged irrespective of the _expunge_ parameter in restoreVirtualMachine API (which was introduced in PR https://github.com/apache/cloudstack/pull/8800 - 4.19.1.0), so no need to check for old root disk exists when expunge is false.
 
https://github.com/apache/cloudstack/blob/2674da2ee9e667ce051b996832c0f3167672688c/engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/VolumeOrchestrator.java#L1187-L1191

- **SSVM / CPVM test issue**
Agent will be in Up state for a while after reboot, so need to wait for sometime for the agent to Disconnect and back to Up. Added wait time after reboot.

Other PR references:
- https://github.com/apache/cloudstack/pull/9591 (4.20.0.0) - Updates hypervisor capabilities and guest os mappings for VMware 80u2 and 80u3.
- https://github.com/apache/cloudstack/pull/9625 (4.20.0.0) - Addresses SystemVM template issues for VMware 80u2 and 80u3.


<!--- Describe your changes in DETAIL - And how has behaviour functionally changed. -->

<!-- For new features, provide link to FS, dev ML discussion etc. -->
<!-- In case of bug fix, the expected and actual behaviours, steps to reproduce. -->

<!-- When "Fixes: #<id>" is specified, the issue/PR will automatically be closed when this PR gets merged -->
<!-- For addressing multiple issues/PRs, use multiple "Fixes: #<id>" -->
<!-- Fixes: # -->

<!--- ******************************************************************************* -->
<!--- NOTE: AUTOMATION USES THE DESCRIPTIONS TO SET LABELS AND PRODUCE DOCUMENTATION. -->
<!--- PLEASE PUT AN 'X' in only **ONE** box -->
<!--- ******************************************************************************* -->

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

### Feature/Enhancement Scale or Bug Severity

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):

### How Has This Been Tested?

Tested VM/Volume operations manually, and smoke tests.

<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, and the tests you ran to -->

#### How did you try to break this feature and the system with this change?

<!-- see how your change affects other areas of the code, etc. -->

<!-- Please read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/main/CONTRIBUTING.md) document -->

sureshanaparti commented:
@blueorangutan package

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
@sureshanaparti a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

weizhouapache commented on plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java:
```diff
@@ -2715,8 +2715,12 @@ private boolean powerOnVM(final VirtualMachineMO vmMo, final String vmInternalCS
+                                e.getMessage().contains("Unable to access file") ||
+                                e.getMessage().contains("it is locked"))) {
                     logger.debug(String.format("Failed to power on VM %s with hostname %s. Retrying", vmInternalCSName, vmNameOnVcenter));
+                    Thread.sleep(1000);
```
would it be better to surround line 2723 by try-catch ?

weizhouapache reviewed (approved):
lgtm

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 12828

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
@sureshanaparti a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u2

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u1

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u1) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian test result (tid-12768)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 78805 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12768-vmware-80u3.zip
Smoke tests completed. 134 look OK, 7 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_deploy_more_vms_than_limit_allows | `Error` | 223.40 | test_deploy_vms_in_parallel.py
test_03_deploy_and_scale_kubernetes_cluster | `Failure` | 102.81 | test_kubernetes_clusters.py
test_05_basic_lifecycle_kubernetes_cluster | `Error` | 88.31 | test_kubernetes_clusters.py
test_06_delete_kubernetes_cluster | `Failure` | 255.87 | test_kubernetes_clusters.py
test_reboot_router | `Error` | 459.86 | test_network.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1521.98 | test_network.py
test_02_restore_vm_with_disk_offering | `Error` | 104.27 | test_restore_vm.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 60.25 | test_restore_vm.py
test_01_snapshot_to_volume | `Error` | 9.37 | test_snapshots.py
test_08_reboot_cpvm | `Failure` | 39.37 | test_ssvm.py
test_01_volume_usage | `Error` | 95.45 | test_usage.py

blueorangutan commented:
<b>[SF] Trillian test result (tid-12771)</b>
Environment: vmware-80u1 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 76196 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12771-vmware-80u1.zip
Smoke tests completed. 136 look OK, 5 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_09_connectivity_between_network_and_vpc_tier | `Failure` | 34.56 | test_ipv4_routing.py
test_11_isolated_network_with_dynamic_routed_mode | `Failure` | 163.88 | test_ipv4_routing.py
test_12_vpc_and_tier_with_dynamic_routed_mode | `Failure` | 411.53 | test_ipv4_routing.py
test_01_ping_in_vr_success | `Failure` | 60.52 | test_diagnostics.py
test_04_autoscale_kubernetes_cluster | `Failure` | 158.03 | test_kubernetes_clusters.py
test_05_basic_lifecycle_kubernetes_cluster | `Failure` | 800.65 | test_kubernetes_clusters.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1524.38 | test_network.py
test_02_restore_vm_with_disk_offering | `Error` | 58.20 | test_restore_vm.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 114.73 | test_restore_vm.py

weizhouapache commented:
@blueorangutan package

blueorangutan commented:
@weizhouapache a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✖️ el9 ✔️ debian ✖️ suse15. SL-JID 12844

blueorangutan commented:
<b>[SF] Trillian test result (tid-12770)</b>
Environment: vmware-80u2 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 80994 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12770-vmware-80u2.zip
Smoke tests completed. 134 look OK, 7 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
ContextSuite context=TestClusterDRS>:setup | `Error` | 0.00 | test_cluster_drs.py
test_02_internallb_roundrobin_1RVPC_3VM_HTTP_port80 | `Failure` | 488.47 | test_internal_lb.py
test_02_internallb_roundrobin_1RVPC_3VM_HTTP_port80 | `Error` | 488.49 | test_internal_lb.py
test_03_vpc_internallb_haproxy_stats_on_all_interfaces | `Error` | 123.15 | test_internal_lb.py
test_03_vpc_internallb_haproxy_stats_on_all_interfaces | `Error` | 123.16 | test_internal_lb.py
test_04_rvpc_internallb_haproxy_stats_on_all_interfaces | `Error` | 331.55 | test_internal_lb.py
test_04_rvpc_internallb_haproxy_stats_on_all_interfaces | `Error` | 331.58 | test_internal_lb.py
test_09_connectivity_between_network_and_vpc_tier | `Error` | 149.42 | test_ipv4_routing.py
test_08_upgrade_kubernetes_ha_cluster | `Failure` | 3692.24 | test_kubernetes_clusters.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1524.99 | test_network.py
test_02_restore_vm_with_disk_offering | `Error` | 60.36 | test_restore_vm.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 102.21 | test_restore_vm.py
test_01_volume_usage | `Error` | 88.37 | test_usage.py

Pearl1594 commented:
@blueorangutan test ol8 vmware-80u1

Pearl1594 commented:
@blueorangutan test ol8 vmware-80u2

blueorangutan commented:
@Pearl1594 a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u2) has been kicked to run smoke tests

Pearl1594 commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@Pearl1594 a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-12803)<b/>

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-12804)<b/>

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-12833)<b/>

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
@sureshanaparti a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 12908

weizhouapache commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@weizhouapache a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-12837)<b/>

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 12920

weizhouapache commented:
@blueorangutan test r8 vmware-80u3

blueorangutan commented:
@weizhouapache [SL] unsupported parameters provided. Supported mgmt server os are: `ol8, ol9, debian12, rocky8, alma9, suse15, centos7, centos6, alma8, ubuntu18, ubuntu22, ubuntu20, ubuntu24`. Supported hypervisors are: `kvm-centos6, kvm-centos7, kvm-rocky8, kvm-ol8, kvm-ol9, kvm-alma8, kvm-alma9, kvm-ubuntu18, kvm-ubuntu20, kvm-ubuntu22, kvm-ubuntu24, kvm-debian12, kvm-suse15, vmware-55u3, vmware-60u2, vmware-65u2, vmware-67u3, vmware-70u1, vmware-70u2, vmware-70u3, vmware-80, vmware-80u1, vmware-80u2, vmware-80u3, xenserver-65sp1, xenserver-71, xenserver-74, xenserver-84, xcpng74, xcpng76, xcpng80, xcpng81, xcpng82, xcpng83`

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian test result (tid-12848)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 71157 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12848-vmware-80u3.zip
Smoke tests completed. 139 look OK, 2 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1522.69 | test_network.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 77.09 | test_restore_vm.py

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 13005

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u2

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u2

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u2) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-12948)<b/>

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian test result (tid-12935)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 84851 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12935-vmware-80u3.zip
Smoke tests completed. 133 look OK, 8 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
ContextSuite context=TestClusterDRS>:setup | `Error` | 0.00 | test_cluster_drs.py
test_03_create_vpc_domain_vpc_offering | `Error` | 26.37 | test_domain_vpc_offerings.py
test_08_upgrade_kubernetes_ha_cluster | `Failure` | 32.17 | test_kubernetes_clusters.py
test_10_vpc_tier_kubernetes_cluster | `Error` | 24.11 | test_kubernetes_clusters.py
test_11_test_unmanaged_cluster_lifecycle | `Error` | 30.34 | test_kubernetes_clusters.py
ContextSuite context=TestListVolumes>:setup | `Error` | 0.00 | test_list_volumes.py
test_list_system_vms_metrics_history | `Failure` | 0.32 | test_metrics_api.py
ContextSuite context=TestNestedVirtualization>:setup | `Error` | 0.00 | test_nested_virtualization.py
ContextSuite context=TestNetworkACL>:setup | `Error` | 0.00 | test_network_acl.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1521.98 | test_network.py

blueorangutan commented:
<b>[SF] Trillian test result (tid-12949)</b>
Environment: vmware-80u2 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 72429 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12949-vmware-80u2.zip
Smoke tests completed. 140 look OK, 1 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1523.36 | test_network.py

blueorangutan commented:
<b>[SF] Trillian test result (tid-12951)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 76520 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12951-vmware-80u3.zip
Smoke tests completed. 139 look OK, 2 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_deploy_more_vms_than_limit_allows | `Error` | 296.87 | test_deploy_vms_in_parallel.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1523.92 | test_network.py

DaanHoogland commented:
@sureshanaparti , good work. this test_network error seems concequent. Was that present before?

DaanHoogland commented:
@blueorangutan test ol8 vmware-70u3

blueorangutan commented:
@DaanHoogland a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-70u3) has been kicked to run smoke tests

sureshanaparti commented:
> @sureshanaparti , good work. this test_network error seems concequent. Was that present before?

@DaanHoogland  test_network fails due to the template url "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.ova" returns redirection response (302). I've updated the new url in marvin test data in this PR, seems it's not taking from there.

weizhouapache commented:
> > @sureshanaparti , good work. this test_network error seems concequent. Was that present before?
> 
> @DaanHoogland test_network fails due to the template url "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.ova" returns redirection response (302). I've updated the new url "https://cloud-images.ubuntu.com/releases/jammy/release/ubuntu-22.04-server-cloudimg-amd64.ova" in marvin test data in this PR, seems it's not taking from there. this is failing in other PRs as well.

@sureshanaparti 
I've updated test_data.py.j2 just now. let's rekick the test

blueorangutan commented:
@weizhouapache a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

sureshanaparti commented:
> > > @sureshanaparti , good work. this test_network error seems concequent. Was that present before?
> > 
> > 
> > @DaanHoogland test_network fails due to the template url "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.ova" returns redirection response (302). I've updated the new url "https://cloud-images.ubuntu.com/releases/jammy/release/ubuntu-22.04-server-cloudimg-amd64.ova" in marvin test data in this PR, seems it's not taking from there. this is failing in other PRs as well.
> 
> @sureshanaparti I've updated test_data.py.j2 just now. let's rekick the test

thanks @weizhouapache

blueorangutan commented:
<b>[SF] Trillian test result (tid-12977)</b>
Environment: vmware-70u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 66920 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t12977-vmware-70u3.zip
Smoke tests completed. 138 look OK, 3 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_04_list_domains_level_filter | `Failure` | 0.04 | test_list_domains.py
test_05_list_domains_no_filter | `Failure` | 0.03 | test_list_domains.py
test_05_list_volumes_isrecursive | `Failure` | 0.06 | test_list_volumes.py
test_07_list_volumes_listall | `Failure` | 0.05 | test_list_volumes.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:setup | `Error` | 1523.16 | test_network.py

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u2

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u2) has been kicked to run smoke tests

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian test result (tid-13092)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 74461 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t13092-vmware-80u3.zip
Smoke tests completed. 139 look OK, 2 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_deploy_more_vms_than_limit_allows | `Error` | 225.44 | test_deploy_vms_in_parallel.py
test_01_deployVMInSharedNetwork | `Error` | 268.78 | test_network.py

blueorangutan commented:
<b>[SF] Trillian test result (tid-13091)</b>
Environment: vmware-80u2 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 74879 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t13091-vmware-80u2.zip
Smoke tests completed. 139 look OK, 2 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_01_internallb_roundrobin_1VPC_3VM_HTTP_port80 | `Error` | 633.43 | test_internal_lb.py
test_01_internallb_roundrobin_1VPC_3VM_HTTP_port80 | `Error` | 633.45 | test_internal_lb.py
test_01_deployVMInSharedNetwork | `Error` | 203.79 | test_network.py

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
@sureshanaparti a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✖️ debian ✔️ suse15. SL-JID 13225

weizhouapache commented:
@blueorangutan package

blueorangutan commented:
@weizhouapache a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 13226

rohityadavcloud commented:
@blueorangutan package

blueorangutan commented:
@rohityadavcloud a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 13298

rohityadavcloud commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
@rohityadavcloud a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u3) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian test result (tid-13243)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 87874 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t13243-vmware-80u3.zip
Smoke tests completed. 137 look OK, 4 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
ContextSuite context=TestClusterDRS>:setup | `Error` | 0.00 | test_cluster_drs.py
test_deploy_more_vms_than_limit_allows | `Error` | 217.70 | test_deploy_vms_in_parallel.py
test_01_deployVMInSharedNetwork | `Failure` | 3624.54 | test_network.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:teardown | `Error` | 3625.85 | test_network.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 73.04 | test_restore_vm.py

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
@sureshanaparti a [SL] Jenkins job has been kicked to build packages. It will be bundled with  KVM, XenServer and VMware SystemVM templates. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 13381

DaanHoogland commented:
@blueorangutan test ol8 vmware-80u3

blueorangutan commented:
<b>[SF] Trillian test result (tid-13307)</b>
Environment: vmware-80u3 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 122502 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t13307-vmware-80u3.zip
Smoke tests completed. 137 look OK, 4 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_08_upgrade_kubernetes_ha_cluster | `Failure` | 3711.40 | test_kubernetes_clusters.py
test_01_deployVMInSharedNetwork | `Error` | 414.13 | test_network.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 67.66 | test_restore_vm.py
test_01_migrate_vm_strict_tags_success | `Error` | 3607.88 | test_vm_strict_host_tags.py
test_02_migrate_vm_strict_tags_failure | `Error` | 4.12 | test_vm_strict_host_tags.py
test_01_restore_vm_strict_tags_success | `Error` | 23.90 | test_vm_strict_host_tags.py
test_02_restore_vm_strict_tags_failure | `Error` | 3604.35 | test_vm_strict_host_tags.py
test_01_scale_vm_strict_tags_success | `Error` | 24.86 | test_vm_strict_host_tags.py
test_02_scale_vm_strict_tags_failure | `Error` | 3607.30 | test_vm_strict_host_tags.py
test_01_deploy_vm_on_specific_host_without_strict_tags | `Error` | 23.42 | test_vm_strict_host_tags.py
test_02_deploy_vm_on_any_host_without_strict_tags | `Error` | 3605.85 | test_vm_strict_host_tags.py
test_03_deploy_vm_on_specific_host_with_strict_tags_success | `Error` | 6.14 | test_vm_strict_host_tags.py
test_04_deploy_vm_on_any_host_with_strict_tags_success | `Error` | 6.03 | test_vm_strict_host_tags.py

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u2

sureshanaparti commented:
@blueorangutan test ol8 vmware-80u1

blueorangutan commented:
@sureshanaparti a [SL] Trillian-Jenkins test job (ol8 mgmt + vmware-80u1) has been kicked to run smoke tests

blueorangutan commented:
<b>[SF] Trillian Build Failed (tid-13329)<b/>

vladimirpetrov reviewed (approved):
LGTM based on manual testing.

blueorangutan commented:
<b>[SF] Trillian test result (tid-13328)</b>
Environment: vmware-80u2 (x2), Advanced Networking with Mgmt server ol8
Total time taken: 75117 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr10586-t13328-vmware-80u2.zip
Smoke tests completed. 137 look OK, 4 have errors, 0 did not run
Only failed and skipped tests results shown below:


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_deploy_more_vms_than_limit_allows | `Error` | 196.18 | test_deploy_vms_in_parallel.py
test_01_deployVMInSharedNetwork | `Failure` | 3604.36 | test_network.py
ContextSuite context=TestSharedNetworkWithConfigDrive>:teardown | `Error` | 3605.64 | test_network.py
test_03_restore_vm_with_disk_offering_custom_size | `Error` | 57.38 | test_restore_vm.py
test_01_scale_vm | `Error` | 1.53 | test_scale_vm.py
test_02_scale_vm_negative_offering_disable_scaling | `Error` | 1.49 | test_scale_vm.py
test_03_scale_vm_negative_vm_disable_scaling | `Error` | 1.46 | test_scale_vm.py
